isis/src/base/objs/PvlToJSON/PvlToJSON.cpp

```diff
@@ -349,4 +351,24 @@ namespace Isis {
     return pvlObjectToJSON(pvl);
   }
 
+  /**
+   * Converts an PVL file to a json object.
+   *
+   *
+   * @param pvlFile Path to an PVL file.
+   *
+   * @return json The pvl file converted to a json object.
+   */
+  json pvlToJSON(QString pvlFile) {
+    Pvl pvl(pvlFile);
+    QFile file(pvlFile);
+
+    if (!file.open(QIODevice::ReadOnly)) {
+      QString message = QString("Failed to open file for PVL Input: [%1]").arg(pvlFile);
+      throw IException(IException::Io, message, _FILEINFO_);
+    }
+    file.close();
+
+    return pvlToJSON(pvl);
```

Comment:
Looking over the Pvl constructor from a QString, it takes care of the majority of the necessary tests on the file. I think we can just wrap line 363 in a try catch block then just rethrow the error.